src/databricks/labs/ucx/runtime.py

```diff
@@ -67,6 +69,37 @@ def inventorize_mounts(cfg: MigrationConfig):
     mounts.inventorize_mounts()
 
 
+@task("assessment", depends_on=[inventorize_mounts, crawl_tables])
+def inventorize_external_locations(cfg: MigrationConfig):
+    """In this part of the assessment, we're going to look up the location of all the tables. Locations that use
+    Mount Points will be looked up. We will then find out the External Locations required for a proper migration and
+    save these to a Table.
+    """
+    ws = WorkspaceClient(config=cfg.to_databricks_config())
+    assess = AssessmentToolkit(ws, "hive_metastore", cfg.inventory_database, RuntimeBackend())
+    assess.generate_ext_loc_list()
+
+
+@task("assessment", depends_on=[setup_schema])
+def inventorize_jobs(cfg: MigrationConfig):
+    """This part scan through all the jobs and locate ones that are not compatible with UC. It looks for common culprits,
+    such as incompatible DBR version, passthru authentication and others. All the jobs are persisted to the jobs tables.
```

Comment:
can you elaborate more instead of `and others`? :) also, use the `$inventory` placeholder to have it replaced with the real database name upon installation